### ISSUE TITLE
error when installing addon

### ISSUE THREAD

**Issue by laantorchaweb** [stefanpenner/ember-moment#91]
Hi, I've used this addon in the past with no problem, but today I've tried to install it on a project and it threw an error, now I've created a new ember cli project and installed the addon, this is the output:

```
➜  testApp git:(master) ✗ ember install ember-moment
version: 1.13.5
Installed packages for tooling via npm.
installing ember-moment
  install addon ember-cli-moment-shim
Installed packages for tooling via npm.
Cannot read property 'reduce' of undefined
TypeError: Cannot read property 'reduce' of undefined
    at Class.module.exports.Task.extend.installBlueprint (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/lib/tasks/addon-install.js:54:24)
    at Class.<anonymous> (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/lib/tasks/addon-install.js:43:19)
    at lib$rsvp$$internal$$tryCatch (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:476:11)
    at lib$rsvp$asap$$flush (/Users/sebastianb/Projects/testApp/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)
    at process._tickCallback (node.js:355:11)
➜  testApp git:(master) ✗
```

do you have any idea about what the error could be?

**Comment by stefanpenner** [comment]
> version: 1.13.5

looks older, you should likely upgrade. I recall something related being fixed.

**Comment by jasonmit** [comment]
Yes, was an issue in ember-cli 1.13.5 that was fixed in 1.13.6
